ui/imports/shared/controls/PasswordInput.qml

```diff
@@ -0,0 +1,31 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+Input { // TODO replace with StatusInput as soon as it supports password
```

Comment:
hmm I guess this refers to `StatusInput` component from `StatusQ` (see `ui/StatusQ/src/StatusQ/Controls/StatusInput.qml`). That said, not sure if we need a separate `PasswordInput` component or the same `StatusInput` could be adapted, could you please have a look?

Reply:
`StatusInput` doesn't support `echoMode` at the moment because it's fundamental type doesn't support it. Essentially, `StatusBaseInput` needs to be redone as `TextEdit`(?) I believe.

It's definitely a bigger undertaking, considering `StatusBaseInput`'s functionality, if we want to ensure we don't break anything.

Reply:
At first I tried to use `StatusInput` but as @PascalPrecht mentioned, its fundamental type (`TextEdit`) doesn't support 'echoMode' straight away. Maybe we should follow Qt's convention and introduce `StatusInputField` for single line input with `TextInput` as its fundamental type. The second option is to rework `StatusInput`. This way we wouldn't need to implement `echoMode` logic ourselves.

Reply:
The idea would be to rework `StatusInput`, this is also what the TODO says: "_TODO replace with StatusInput as soon as it supports password_" but this is a different task. In this case I would implement this task using just another `Input` (as it was) and leave the `TODO` as is. As it is implemented now, a new component is introduced in shared/controls, which is not the idea since such a component should leave in StatusQ and be demonstrated in sandbox app. Additionally, we do not know yet if a new separate component will be needed for password input or the same `StatusInput` can be used in password mode, I would say introducing another component at this point it does not make much sense. Please let me know what you think!

Reply:
Yes, it exists here: https://github.com/status-im/StatusQ/issues/393